lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -1274,12 +1270,10 @@ def install_local(
         try:
             return c[snap_name]
         except SnapAPIError as e:
-            logger.error(
-                "Could not find snap {} when querying Snapd socket: {}".format(snap_name, e.body)
-            )
-            raise SnapError("Failed to find snap {} in Snap cache".format(snap_name))
+            logger.error(f"Could not find snap {snap_name} when querying Snapd socket: {e.body}")
```

Comment:
For logging calls, we should use `%s` and args, not the `.format` that was there before, or f-strings.